app/uk/gov/hmrc/merchandiseinbaggage/service/CalculationService.scala

```diff
@@ -29,13 +29,15 @@ import uk.gov.hmrc.merchandiseinbaggage.utils.DataModelEnriched._
 class CalculationService @Inject()(mibConnector: MibConnector)(implicit ec: ExecutionContext) {
   private val logger = Logger("CalculationService")
 
-  def paymentCalculation(declarationGoods: DeclarationGoods)(implicit hc: HeaderCarrier): Future[PaymentCalculations] =
+  def paymentCalculation(declarationGoods: DeclarationGoods)(implicit hc: HeaderCarrier): Future[PaymentCalculations] = {
```

Comment:
We could actually change this to accept a ```Seq[ImportGoods]``` as we only ever do the Calculation for Import Journey, but this is fine for now I think

Reply:
yes makes sense....i do it now

Reply:
done